images/patroni/entrypoint.sh

```diff
@@ -10,13 +10,26 @@ fi
 cat > /home/postgres/patroni.yml <<__EOF__
 bootstrap:
   dcs:
-    loop_wait: ${PATRONI_LOOP_WAIT}
-    ttl: ${PATRONI_TTL}
-    retry_timeout: ${PATRONI_RETRY_TIMEOUT}
-    master_start_timeout: ${PATRONI_MASTER_START_TIMEOUT}
-    maximum_lag_on_failover: ${PATRONI_MAXIMUM_LAG_ON_FAILOVER}
     postgresql:
       use_pg_rewind: true
+    parameters:
+      shared_buffers: '32GB'
```

Comment:
Why hardcode? My approach used an [env variable](https://github.com/cncf/devstats-docker-images/blob/master/images/patroni/entrypoint.sh#L46) which value was set in the stateful set [here](https://github.com/cncf/devstats-helm/blob/master/devstats-helm/templates/devstats-postgres-statefulset.yaml#L143-L144) using [values.yaml](https://github.com/cncf/devstats-helm/blob/master/devstats-helm/values.yaml#L300) from the Helm chart (in this case 96G).